cmd/kconf.go

```diff
@@ -111,9 +111,13 @@ var listCmd = &cobra.Command{
 		if err != nil {
 			log.Fatal().Msgf("Could not read main config")
 		}
-		contexts := config.List()
+		contexts, currentContext := config.List()
 		for _, ctx := range contexts {
-			fmt.Println(ctx)
+			if currentContext == ctx {
+				fmt.Println("*", ctx)
```

Comment:
This is nitpicky but can you use two-character width for this? Just so there's a space between the `*` and the name, and the other names are aligned by two spaces (like git does).

Reply:
`Println()` function, when args more than one, it auto add a space
https://github.com/golang/go/blob/52c4488471ed52085a29e173226b3cbd2bf22b20/src/fmt/print.go#L1168

Reply:
Ohh, my bad, I didn't even realize it was `Println`